package.json

```diff
@@ -10,23 +10,22 @@
     "type": "opencollective",
     "url": "https://opencollective.com/unified"
   },
+  "type": "module",
   "main": "dist/index.js",
-  "module": "dist/react-remark.esm.js",
   "typings": "dist/index.d.ts",
   "sideEffects": false,
   "files": [
     "dist",
     "src"
   ],
   "engines": {
-    "node": ">=10"
+    "node": ">=12"
   },
   "scripts": {
-    "start": "tsdx watch",
-    "build": "tsdx build",
-    "test": "tsdx test",
-    "lint": "tsdx lint",
-    "prepare": "tsdx build",
+    "build": "tsc && type-coverage",
+    "test": "node --loader ts-node/esm ./test/index.test.ts",
```

Comment:
I'd recommend to use lite https://github.com/privatenumber/tsx over `ts-node`, types linting can be a seperate task via `tsc`.

Reply:
I agree, though neither will be needed in the end.

The goal is:

> migrate TypeScript files to JavaScript with JSDoc based Typescript